app/scripts/modules/ecs/src/ecsCluster/ecsCluster.read.service.ts

```diff
@@ -2,11 +2,19 @@ import { module } from 'angular';
 
 import { REST } from '@spinnaker/core';
 import { IEcsClusterDescriptor } from './IEcsCluster';
+import {IEcsCapacityProvidersDetails} from "../ecsCluster/IEcsCapacityProvidersDetails";
 
 export class EcsClusterReader {
   public listClusters(): PromiseLike<IEcsClusterDescriptor[]> {
     return REST('/ecs/ecsClusters').get();
   }
+
+  public listDescribeClusters(account: string, region: string): PromiseLike<IEcsCapacityProvidersDetails[]> {
```

Comment:
let's simplify this func name to `DescribeClusters`, unless it's not actually doing that?